src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -328,6 +329,82 @@ public void jobLogic () {
                 return;
             }
         }
+        else if ("true".equals(System.getenv("RUN_E2E"))) {
```

Comment:
Yikes. I'm guessing there's no way to avoid this without touching a bunch of other things?